css/style.css

```diff
@@ -9,400 +15,2256 @@ body {
+    width: 17%;
+    margin-left: 80%;
+  }
+}
+
+@media (max-width: 1000px) {
+  .bag__link {
+    margin-left: 53%;
+  }
+}
+
+@media (max-width: 850px) {
+  .bag__arrow {
+    margin-left: 82%;
+    width: 15%;
+  }
+}
+
+@media (max-width: 800px) {
+  .bag__wight,
+  .bag__button-number,
+  .bag__link-arrow {
+    display: none;
+  }
+}
+
+@media (max-width: 800px) {
+  .bag__pink {
+    width: 500px;
+    height: 600px;
+    left: 0;
+    top: 752px;
+  }
+}
+
+@media (max-width: 800px) {
+  .card {
+    margin: 5%;
+    width: 90%;
+    top: 1287px;
+    height: 430px;
+  }
+}
+
+@media (max-width: 800px) {
+  .bag__title-bold {
+    font-size: 40px;
+    line-height: 41px;
+    left: 10%;
+    width: 350px;
+  }
+}
+
+@media (max-width: 800px) {
+  .bag__title {
+    font-size: 16px;
+    line-height: 18px;
+    left: 10%;
+  }
+}
+
+@media (max-width: 800px) {
+  .bag-text {
+    width: 80%;
+    top: 140px;
+    left: 10%;
+  }
+}
+
+@media (max-width: 800px) {
+  .bag__color {
+    left: 10%;
+    top: 63%;
+    width: 300px;
+  }
+}
+
+@media (max-width: 800px) {
+  span {
+    top: 65%;
+    left: 65%;
+  }
+}
+
+@media (max-width: 800px) {
+  .color-wight,
+  .color-blue,
+  .color-green {
+    width: 50px;
+    height: 50px;
+    margin-right: 5%;
+  }
+}
+
+@media (max-width: 800px) {
+  .color-pink {
+    width: 60px;
+    height: 60px;
+    margin-right: 5%;
+  }
+}
+
+@media (max-width: 800px) {
+  .bag__button {
+    left: 10%;
+    top: 85%;
+    width: 80%;
+  }
+}
+
+@media (max-width: 660px) {
+  .color-wight,
+  .color-blue,
+  .color-green {
+    width: 40px;
+    height: 40px;
+    margin-right: 3%;
+  }
+}
+
+@media (max-width: 660px) {
+  .color-pink {
+    width: 50px;
+    height: 50px;
+    margin-right: 3%;
+  }
+}
+
+@media (max-width: 660px) {
+  .bag__color {
+    top: 65%;
+  }
+}
+
+@media (max-width: 660px) {
+  .bag__pink {
+    left: -5%;
+  }
+}
+
+@media (max-width: 660px) {
+  .bag-text {
+    font-size: 13px;
+    line-height: 15px;
+  }
+}
+
+@media (max-width: 550px) {
+  .bag__pink {
+    width: 400px;
+  }
+}
+
+@media (max-width: 550px) {
+  .bag__button {
+    font-size: 20px;
+  }
+}
+
+@media (max-width: 450px) {
+  .color-pink {
+    width: 35px;
+    height: 35px;
+    margin-right: 2%;
+  }
+}
+
+@media (max-width: 450px) {
+  .color-wight,
+  .color-blue,
+  .color-green {
+    width: 25px;
+    height: 25px;
+    margin-right: 2%;
+  }
+}
+
+@media (max-width: 450px) {
+  .bag__color {
+    top: 68%;
+  }
+}
+
+@media (max-width: 450px) {
+  .bag__title-bold {
+    font-size: 25px;
+    line-height: 27px;
+    width: 270px;
+  }
+}
+
+@media (max-width: 450px) {
+  .bag__pink {
+    width: 300px;
+  }
+}
+
+@media (max-width: 450px) {
+  .card {
+    top: 1250px;
+  }
+}
+
+@media (max-width: 400px) {
+  .bag__button {
+    font-size: 15px;
+  }
+}
+
+@media (max-width: 400px) {
+  .bag-text {
+    font-size: 12px;
+    line-height: 13px;
+  }
+}
+
+@media (max-width: 335px) {
+  .card { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)